src/Core/Scheduler/Providers/WFS_Provider.js

```diff
@@ -4,61 +4,132 @@
  * Description: Provides data from a WFS stream
  */
 
-
+import Extent from '../../Geographic/Extent';
 import Provider from './Provider';
 import Fetcher from './Fetcher';
 import CacheRessource from './CacheRessource';
+import GeoJSON2Three from '../../../Renderer/ThreeExtended/GeoJSON2Three';
 
-/**
- * Return url wmts MNT
- * @param {Object} options
- * @param {?string} options.url - service base url
- * @param {?string} options.layer - requested data layer
- * @param {?string} options.format - image format (default: format/jpeg)
- * @constructor
- */
-// TODO should be refit to adhere the same logic as other providers (ie one instance for all layers)
-function WFS_Provider(options) {
+function WFS_Provider() {
     this.cache = CacheRessource();
-    this.baseUrl = options.url || '';
-    this.networkOptions = options.networkOptions;
-    this.layer = options.layer || '';
-    this.typename = options.typename || '';
-    this.format = options.format === undefined ? 'json' : options.format;
-    this.epsgCode = options.epsgCode || 4326;
+    this.batiments = [];
```

Comment:
See https://github.com/iTowns/itowns/pull/431/files#diff-6e5ed93be78417d38f4b8eb6d6f2a5e1R202
I only keep the geometries which have their 1st point inside the tile that requested the features.

So no duplicate.
This filtering mechanism is basic but it works for my tests data. We can extend / improve it layer (for instance by allowing the user to supply a `filterFeature` function)